packages/firestore/package.json

```diff
@@ -18,7 +18,7 @@
     "dev": "rollup -c -w",
     "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
-    "prettier": "prettier --write '*.js' '{,!(dist)}/**/*.ts'",
+    "prettier": "prettier --write '*.js' '*.ts' '{,!(dist)}/**/*.ts'",
```

Comment:
Question: what's the reason for this addition? Shouldn't the glob on the right cover it?

Reply:
I tried to figure this out with https://globster.xyz/, but it seems to slightly disagree with what we have.  I think `'{,!(dist)}/**/*.ts'` doesn't match since it requires at least one level of nesting.